scripts/upgrade.sh

```diff
@@ -27,29 +27,29 @@ get_k3s_process_info() {
 
   # If we still have multiple pids, print out the matching process info for troubleshooting purposes,
   # and exit with a fatal error.
-  if [ "$(echo $K3S_PID | wc -w)" != "1" ]; then
+   if [ "$(echo "$K3S_PID" | wc -w)" != "1" ]; then
     for PID in $K3S_PID; do
-      ps -fp $PID || true
+      ps -fp "$PID" || true
     done
     fatal "Found multiple K3s pids"
   fi
 
-  K3S_PPID=$(ps -p $K3S_PID -o ppid= | awk '{print $1}')
+  K3S_PPID=$(ps -p "$K3S_PID" -o ppid= | awk '{print $1}')
   info "K3S binary is running with pid $K3S_PID, parent pid $K3S_PPID"
 
   # When running with the --log flag, the 'k3s server|agent' process is nested under a 'k3s init' process.
   # If the parent pid is not 1 (init/systemd) then we are nested and need to operate against that 'k3s init' pid instead.
   # Make sure that the parent pid is actually k3s though, as openrc systems may run k3s under supervise-daemon instead of
   # as a child process of init.
-  if [ "$K3S_PPID" != "1" ] && grep -a k3s /host/proc/${K3S_PPID}/cmdline | grep -q -v supervise-daemon; then
+  if [ "$K3S_PPID" != "1" ] && grep -a k3s /host/proc/"${K3S_PPID}"/cmdline | grep -q -v supervise-daemon; then
```

Comment:
I missed this one earlier
```suggestion
  if [ "$K3S_PPID" != "1" ] && grep -a k3s "/host/proc/${K3S_PPID}/cmdline" | grep -q -v supervise-daemon; then
```